src/services/chris_integration.tsx

```diff
@@ -86,13 +87,13 @@ export const modifyDatetime = (oldDay: string): string => {
   return rvtVal
 }
 
-class ChrisIntegration {
+export const isNotModel = (modelName: string): boolean => { // Dynamically loop through all model plug-ins and check if the current plug-in is valid
```

Comment:
I would suggest making this function into `isModel` and where you need `isNotModel` use `!isModel`.

@janakitti what do you think on this matter?

Reply:
Yeah I'd agree. I think it's more conventional to use the positive form for boolean variable names.